js_modules/dagster-ui/packages/ui-core/src/assets/PartitionHealthSummary.tsx

```diff
@@ -1,73 +1,53 @@
-import {PartitionDimensionSelection, PartitionHealthData} from './usePartitionHealthData';
-import {displayNameForAssetKey} from '../asset-graph/Utils';
+import {PartitionHealthData} from './usePartitionHealthData';
+import {LiveDataForNode, displayNameForAssetKey} from '../asset-graph/Utils';
 import {PartitionStatus} from '../partitions/PartitionStatus';
 
 interface Props {
   assetKey: AssetKey;
   showAssetKey?: boolean;
   data: PartitionHealthData[];
-  selections?: PartitionDimensionSelection[];
+  partitionStats: LiveDataForNode['partitionStats'];
 }
 
 export const PartitionHealthSummary = memo((props: Props) => {
-  const {showAssetKey, assetKey, data, selections} = props;
+  const {showAssetKey, assetKey, data, partitionStats} = props;
   const assetData = data.find(
     (d) => JSON.stringify(d.assetKey.path) === JSON.stringify(assetKey.path),
   );
 
-  if (!assetData) {
-    return (
-      <div style={{minHeight: 55, position: 'relative'}}>
-        <Spinner purpose="section" />
-      </div>
-    );
-  }
-
-  const keysForTotals = selections
-    ? selections.map((r) => r.selectedKeys)
-    : assetData.dimensions.map((d) => d.partitionKeys);
-
-  const total = keysForTotals.reduce((total, d) => d.length * total, 1);
-
-  const success = keysForTotals
-    .reduce(
-      (combinations, d) =>
-        combinations.length
-          ? combinations.flatMap((keys) => d.map((key) => [...keys, key]))
-          : d.map((key) => [key]),
-      [] as string[][],
-    )
-    .filter((dkeys) => assetData.stateForKey(dkeys) === AssetPartitionStatus.MATERIALIZED).length;
-
   return (
     <div>
       <Box flex={{justifyContent: 'space-between'}} style={{fontWeight: 600}} margin={{bottom: 4}}>
         <Caption>{showAssetKey ? displayNameForAssetKey(assetKey) : 'Materialized'}</Caption>
-        <Caption>{`${success.toLocaleString()}/${total.toLocaleString()}`}</Caption>
+        {partitionStats ? (
+          <Caption>{`${partitionStats.numMaterialized.toLocaleString()}/${partitionStats.numPartitions.toLocaleString()}`}</Caption>
+        ) : null}
       </Box>
-      {assetData.dimensions.map((dimension, dimensionIdx) => (
-        <Box key={dimensionIdx} margin={{bottom: 4}}>
-          {assetData.dimensions.length > 1 && <Caption>{dimension.name}</Caption>}
-          <PartitionStatus
-            small
-            partitionNames={dimension.partitionKeys}
-            splitPartitions={!isTimeseriesDimension(dimension)}
-            selected={selections ? selections[dimensionIdx]!.selectedKeys : undefined}
-            health={{
-              ranges: assetData.rangesForSingleDimension(
-                dimensionIdx,
-                selections?.length === 2 ? selections[1 - dimensionIdx]!.selectedRanges : undefined,
-              ),
-            }}
-          />
-        </Box>
-      ))}
+      {!assetData ? (
+        <div style={{minHeight: 55, position: 'relative'}}>
+          <Spinner purpose="section" />
+        </div>
```

Comment:
Just moved the loading state down here to the assetData section so the "Materialized: 100 / 2000" can be displayed above fast.